.github/workflows/cli.yaml

```diff
@@ -48,22 +38,14 @@ jobs:
             - name: Install dependencies
               run: pnpm install
 
-            - name: Test / Coverage
-              working-directory: ./apps/cli
-              run: pnpm run test:coverage
+            - name: Build
+              run: pnpm build --filter @cartesi/cli
+
+            - name: Test
+              run: pnpm test --filter @cartesi/cli
```

Comment:
The next step will require that the test:covarega is executed.

Reply:
Maybe it's better to run inside apps/cli to ensure the coverage is in the right place instead of using --filter ?

Reply:
I enabled the coverage in the vitest config file, so it's always on now